src/implementations/twig/components/site-header/ec-site-header.story.js

```diff
@@ -58,7 +58,8 @@ const getArgs = (data) => {
   }
   if (data.language_selector) {
     defaultArgs.languages_eu = data.language_selector.overlay.items.length;
-    defaultArgs.languages_non_eu = 5;
+    defaultArgs.languages_non_eu =
+      data.language_selector.overlay.non_eu_items.length;
```

Comment:
mmh..but why do we "expose" a default value for this..? It's not anything related to a parameter, right..? we use it to showcase the component with any kind of amounf of items, so the "table" should simply be empty, there is no default value,